### PR TITLE
hash/nes.xml: add Micro Mages to working software list

### PR DESCRIPTION
**Pull request by chungy**
New working software list (nes.xml)
-----------------------------------
Micro Mages
Micro Mages Second Quest

Micro Mages can be purchased in both physical and digital format; http://morphcat.de/micromages/ has links to the store fronts. I've dumped the physical cart and verified it is the same ROM as the digital release. Micro Mages Second Quest is a digital-only release.

I'll attach a photo of the PCB here too, but I don't think it has useful information. If I'm wrong, feel free to use it :)

IMG_20240910_174516

### CODE REVIEW

**Review by cuavas — CHANGES_REQUESTED** [review]
Please indent with one tab per nested element level.  You’ve used a mixture of tabs and spaces for indentation here.

File: hash/nes.xml [hunk 1]
```diff
@@ -24626,6 +24626,43 @@ license:CC0-1.0
...
+		  <rom name="Micro Mages chr" size="8192" crc="24144d94" sha1="e08bec8c93dce07262be37c7cceb7019817bea69" />
+		</dataarea>
+	  </part>
+	</software>
+
+	<software name="micromages2">
+	  <description>Micro Mages Second Quest</description>
+	  <year>2022</year>
+	  <publisher>Morphcat Games</publisher>
+	  <info name="release" value="20220915" />
```
Comment (cuavas) [inline hash/nes.xml:24652]: This is less than three years old, and only available as a paid download.  As such, it should not be added to the software list.  We generally try to be nice to software developers.

**Review by cuavas — CHANGES_REQUESTED** [review]

**Comment by chungy** [comment]
> Please indent with one tab per nested element level. You’ve used a mixture of tabs and spaces for indentation here.

Thanks, didn't notice Emacs was doing this to me :)

File: hash/nes.xml (hunk 1 above)
Comment (chungy) [reply hash/nes.xml]: Makes sense, feel free to reject the merge. The two games come as a bundle.

**Comment by cuavas** [comment]
I added your cartridge dump in 81bc5452e515e5c09751414e54eb8578fd43f07a.

**Comment by chungy** [comment]
Awesome, thanks!

**Pull request closed without merging**